rivington/sass/_config-child-theme-deep.scss

```diff
@@ -166,7 +166,7 @@ $config-elements: (
 
 		// Borders
 		"border": (
-			"width": 0,
+			"width": 1px,
```

Comment:
Question: Should we add this to the custom colors extra css? 

It's minor enough that folks probably won't notice, but on the other hand, we don't _need_ it unless custom colors are active.

Reply:
Ahh, yeah I agree. 
I’ll do this with the extra CSS.